### PR TITLE
{bio,vis}[intel/2020b] NanoComp v1.13.1, NanoPlot v1.33.0, Kaleido v0.1.0, ... w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#11992~~ (Arrow), ~~#11993~~ (Pysam), ~~#11994~~ (Seaborn), ~~#11995~~ (Biopython), ~~#11996~~ (pauvre), ~~#11997~~ (statsmodels)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/f6c3c656eeec283e39748619d752fe91 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node109.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/7edfcef2e8812ed1eb4c46bc2f40e86d for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

File: easybuild/easyconfigs/n/NanoPlot/NanoPlot-1.33.0-intel-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ('statsmodels', '0.12.1'),
+    ('Arrow', '0.17.1'),  # for pyarrow
+    ('Kaleido', '0.1.0'),
+]
+
+download_dep_fail = True
+use_pip = True
+
+# replace strict Seaborn & pauvre version requirement with minimal version requirements
+preinstallopts = "sed -i 's/==0/>=0/g' setup.py && "
```
Comment (lexming) [inline easybuild/easyconfigs/n/NanoPlot/NanoPlot-1.33.0-intel-2020b.eb:41]: `preinstallopts` is redefined here, seems a leftover from the easyconfig of `NanoComp`

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/n/NanoPlot/NanoPlot-1.33.0-intel-2020b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/n/NanoPlot/NanoPlot-1.33.0-intel-2020b.eb:37]: fixed in f999a60b53

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node3504.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/6d4a84697d7a67f3a151fc9a4866c9e4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node2606.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/644f5a5cc21635768116c78ba82ad7dc for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/9edef9dbba05398f1e05908cf21a5b3c for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**